modules/psap-node-feature-discovery-topology-updater-command-reference.adoc

```diff
@@ -0,0 +1,190 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_content-type: REFERENCE
+[id="nfd-topology-updater-command-line-flags_{context}"]
+= NFD Topology Updater command line flags
+
+[discrete]
```

Comment:
Did you consider adding these flags as a table? I know it's sometimes hard to get all that into a small table, but that's usually how I see flags presented.